src/utils/exportUtils/Exporter.ts

```diff
@@ -80,17 +80,30 @@ export default abstract class Exporter {
     protected async downloadZIP(): Promise<string | void> {
         const filename = this.destinationFileName;
         const filenameWithoutExt = filename.substring(0, filename.lastIndexOf(".")); // take off the extension
-        const { default: JSZip } = await import("jszip");
+        const { BlobWriter, ZipWriter } = await import("@zip.js/zip.js");
 
-        const zip = new JSZip();
+        const zipFileWriter = new BlobWriter("application/zip");
+        const zipWriter = new ZipWriter(zipFileWriter, { zip64: true });
         // Create a writable stream to the directory
         if (!this.cancelled) this.updateProgress(_t("export_chat|generating_zip"));
         else return this.cleanUp();
 
-        for (const file of this.files) zip.file(filenameWithoutExt + "/" + file.name, file.blob);
+        // Sometimes, media with a duplicate file name is written by an exporter.
+        // This will be reflected by the logger statement in the loop below.
+        // This should probably be fixed properly, by somehow differentiating these files by date.
+        // In the meantime, let's just pick the "last added file" as the preferred option in these cases, and log the errors in all other cases.
+        // If es2023 were to be the build target, this could become `toReversed()` to be cleaner.
+        for (const file of this.files.reverse()) {
+            const filename = filenameWithoutExt + "/" + file.name;
+            try {
+                await zipWriter.add(filename, file.blob.stream(), {zip64: true});
```

Comment:
Docs say zip64 is automatically set to true when necessary so leaving it as the default might be a better option.